### PR TITLE
Change implicit to using

### PR DESCRIPTION
**Pull request by odersky**

### CODE REVIEW

**Comment by odersky** [comment]
test performance please

File: compiler/src/dotty/tools/dotc/classpath/ZipAndJarFileLookupFactory.scala [hunk 1]
```diff
@@ -10,8 +10,9 @@ import java.nio.file.attribute.{BasicFileAttributes, FileTime}
 
 import scala.annotation.tailrec
 import dotty.tools.io.{AbstractFile, ClassPath, ClassRepresentation, FileZipArchive, ManifestResources}
-import dotty.tools.dotc.core.Contexts.Context
+import dotty.tools.dotc.core.Contexts.{Context, ctx}
 import FileUtils._
+import dotty.tools.dotc.core.Contexts.{Context, ctx}
```
Comment (soronpo) [inline compiler/src/dotty/tools/dotc/classpath/ZipAndJarFileLookupFactory.scala:15]: This import is a duplicate of the one two lines above it.

Comment (odersky) [reply compiler/src/dotty/tools/dotc/classpath/ZipAndJarFileLookupFactory.scala:15]: Fixed, thank you!

**Comment by odersky** [comment]
test performance please

**Comment by odersky** [comment]
test performance please

**Comment by odersky** [comment]
I am going to merge this ASAP. I believe it's far too large and uninteresting to review, and it will get out of sync very quickly.

**Pull request merged**